agents-for-bedrock/use-case-examples/hr-assistant/agent/api-schema/calendar-action-group.json

```diff
@@ -0,0 +1,97 @@
+{
```

Comment:
any reason to create the action groups via API schema? Function definition is usually easier for customers to understand